src/controllers/accounts.controller.js

```diff
@@ -42,8 +44,7 @@ exports.register = async (req, res, next) => {
   // check whether inputs are valid
   const validation = validationResult(req);
   if (!validation.isEmpty()) {
-    const msgs = validation.errors.map((err) => `The ${err.param} has incorrect format.`);
-    return res.status(422).json({success: false, errors: msgs});
+    return res.status(422).json(responseMsg.validationError422(validation.errors));
```

Comment:
the msgs returned by `express-validator` are only `Invalid value`. I cannot print it on the screen directly looool
(So originally, I modified it to `The ${err.param} has incorrect format.` at line 45)
maybe map the error message in function `validationError422 ` ? 

![image](https://user-images.githubusercontent.com/5423351/61602737-cc5eee00-ac08-11e9-858b-b3b10b70d1d3.png)

Reply:
This is fixed in #16